submissions/UA-girl/task_html-css-popup/index.html

```diff
@@ -0,0 +1,207 @@
+                                        <img src="img/files.png" alt="Files">
+                                        <span>Files</span>
+                                    </a>
+                                </li>
+                                <li class="nav-item nav-item__inner">
+                                    <a href="#">
+                                        <img src="img/contacts.png" alt="Contacts">
+                                        <span>Contacts</span>
+                                    </a>
+                                </li>
+                                <li class="nav-item nav-item__inner">
+                                    <a href="#">
+                                        <img src="img/tanslation.png" alt="Translate">
+                                        <span>Translate</span>
+                                    </a>
+                                </li>
+                                <li class="nav-item nav-item__inner">
+                                    <a href="#">
+                                        <img src="img/videos.png" alt="Video">
+                                        <span>Videos</span>
+                                    </a>
+                                </li>
+                                <li class="nav-item nav-item__inner">
+                                    <a href="#">
+                                        <img src="img/search.png" alt="Search">
+                                        <span>Search</span>
+                                    </a>
+                                </li>
+                                <li class="nav-item nav-item__inner">
+                                    <a href="#">
+                                        <img src="img/photos.png" alt="Photo">
+                                        <span>Photos</span>
+                                    </a>
+                                </li>
+                                <li class="nav-item nav-item__inner">
+                                    <a href="#">
+                                        <img src="img/apps.png" alt="App">
+                                        <span>Apps</span>
+                                    </a>
+                                </li>
+                                <li class="nav-item nav-item__inner">
+                                    <a href="#">
+                                        <img src="img/documents.png" alt="Documents">
+                                        <span>Documents</span>
+                                    </a>
+                                </li>
+                                <li class="nav-item nav-item__inner__label">
+                                    <input type="checkbox" class="checkbox checkbox__additional hidden"
+                                           id="popup-additional">
+                                    <label for="popup-additional" class="label_popup__additional">
```

Comment:
If I understand you correctly you use BEM notation: block_element__modificator, and in this case, `.label_popup__additional` is class with modificator. In BEM methodology class with modificator can'texist on its own, he must be provided by original class without modificator. [Read more](https://ru.bem.info/methodology/quick-start/#%D0%9C%D0%BE%D0%B4%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%82%D0%BE%D1%80)